cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -66,10 +65,10 @@
     private static final String SERVICE_NAME_SUFFIX = NAME_SUFFIX + "-bootstrap";
     private static final String HEADLESS_SERVICE_NAME_SUFFIX = NAME_SUFFIX + "-brokers";
 
-    private static final String CLIENTS_CA_SUFFIX = NAME_SUFFIX + "-clients-ca";
-    private static final String BROKERS_INTERNAL_SUFFIX = NAME_SUFFIX + "-brokers-internal";
-    private static final String BROKERS_CLIENTS_SUFFIX = NAME_SUFFIX + "-brokers-clients";
-    private static final String CLIENTS_PUBLIC_KEY_SUFFIX = NAME_SUFFIX + "-cert";
+    private static final String BROKERS_SECRET_SUFFIX = NAME_SUFFIX + "-brokers";
+    private static final String CLUSTER_PUBLIC_KEY_SUFFIX = "-cert";
+    private static final String CLIENTS_CA_SUFFIX = "-clients-ca";
+    private static final String CLIENTS_PUBLIC_KEY_SUFFIX = "-clients-ca-cert";
```

Comment:
They are all suffix for Secrets but just the first one has `SECRET` in the name. Do we want to add it to the others or removing from the first one to have consistent naming ?